crypto/README.md

```diff
@@ -11,6 +11,7 @@ This crate contains modules with cryptographic operations needed in STARK proof
 ### Rescue hash function implementation
 Rescue hash function is implemented according to the Rescue Prime [specifications](https://eprint.iacr.org/2020/1143.pdf) with the following exception:
 * We set the number of rounds to 7, which implies a 40% security margin instead of the 50% margin used in the specifications (a 50% margin rounds up to 8 rounds). The primary motivation for this is that having the number of rounds be one less than a power of two simplifies AIR design for computations involving the hash function.
+* For instantiation `RP64_256`, we use the first 4 elements of the state (rather than the last 4 elements of the sate) for capacity and the remaining 8 elements for rate. The output of the hash function comes from the first four elements of the rate portion of the state (elements 4, 5, 6, and 7).
```

Comment:
This probably should enjoy a comment on security.

We are effectively applying a fixed bit-permutation P before and after the f_XLIX permutation. The results will not match what happens when P is the identity, but we assert without proof that if f_XLIX is indistinguishable from a random permutation then  P^{-1} f_XLIX P should likewise be indistinguishable.

Reply:
Makes sense! I've included a slightly simplified version of your text in the comments. Thank you!